recipes/server_debian.rb

```diff
@@ -30,3 +30,10 @@
   supports :restart => true, :status => true, :reload => true
   action [:enable, :start]
 end
+
+unless ::File.directory?('/etc/postgresql/' + node['postgresql']['version'] + '/main')
```

Comment:
Thanks. @sinfomicien can you review @martinisoft's suggestion and update or respond accordingly. I would love to merge this soon.